scripts/ci-e2e.sh

```diff
@@ -58,7 +84,27 @@ source "${M3_DEV_ENV_PATH}/lib/ironic_basic_auth.sh"
 # shellcheck disable=SC1091,SC1090
 source "${M3_DEV_ENV_PATH}/lib/ironic_tls_setup.sh"
 
-# run e2e tests 
+# CAPM3 and CAPI release path
+CAPM3RELEASEPATH="${CAPM3RELEASEPATH:-https://api.github.com/repos/${CAPM3_BASE_URL:-metal3-io/cluster-api-provider-metal3}/releases}"
+CAPIRELEASEPATH="${CAPIRELEASEPATH:-https://api.github.com/repos/${CAPI_BASE_URL:-kubernetes-sigs/cluster-api}/releases}"
+
+# CAPI releases
+export CAPI_TO_RELEASE="${CAPI_TO_RELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v1.1.")}"
+if [ "${CAPI_VERSION}" == "v1alpha3" ]; then
+  export CAPI_FROM_RELEASE="${CAPI_FROM_RELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v0.3.")}"
+else
```

Comment:
I guess this should be 
```suggestion
else if [ "${CAPI_VERSION}" == "v1alpha4" ]; then
```

Reply:
Fixed it in latest commit, kindly check.